### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by joonasviljakainen** [uberballo/MapGenerator#1]
ZIP downloaded on 23.8. at 17:45

Very cool project. Great scope and it's nice to see a visualization. I had some issues starting the application - I assume we use different JDKs. I added openJDK as a dependency to make get the project to launch.

All tests pass, no problem. Great coverage.

__Some thoughts:__

- The code is clean and relatively easy to read. ClassicNoise.java could use some more inline comments, as you've done in OpenSimplexNoise.java.

- Classes have logical structures and no dependencies. 

- The method openNoise is super long. I wonder if it could be chopped up a bit? For example, lines 108-124 do very similar things using variables that aren't referred to anywhere else. Same thing wt lines 173-184. In addition,  the section from line 127 onward ("the magic happens here") appears as if it could be separated into its own method. Or that's what I'd do.

In summary: the project is almost ready to be handed in as-is.
